app/Models/User.php

```diff
@@ -540,7 +540,7 @@ public function rank()
 
     public function rankHistories()
     {
-        return $this->hasMany(RankHistory::class);
+        return $this->hasMany("App\Models\RankHistory", 'user_id', 'user_id');
```

Comment:
the relation between user and rankhistories was broken, so I changed the statement to match how all the other relations are defined. is this ok?